deps/querystring.lua

```diff
@@ -25,84 +25,89 @@ limitations under the License.
   tags = {"luvit", "url", "codec"}
 ]]
 
-local find = string.find
```

Comment:
Why remove the local definitions?

Reply:
For consistency with other modules

Reply:
There's not much consistency here. Some modules do this, some don't.

Reply:
In that case, there's not much reason to do it one way or the other besides personal preference. I prefer it without the local definitions, but if you want them, lemme know

Reply:
use `local` for faster.